src/XMODULE.cpp

```diff
@@ -7,6 +7,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 #include "cnglob.h"
+#if defined(SUPPORT_XMODULE)
```

Comment:
I don't think these are necessary either. If XMODULE isn't part of the CMake executable, these won't do anything.